.prettierrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
```

Comment:
Prettier rule to apply. Let me know your preferences we would be able to add more

Reply:
Up to you, as long as it is fine with everyone.